src/components/ArticlePage/index.js

```diff
@@ -95,6 +127,21 @@ export class ArticlePage extends React.Component {
     });
   }
 
+  identifyUserBookmarks() {
+    const { bookmarks, match } = this.props;
+    const { bookmarkedArticles } = bookmarks;
+    const currentViewedArticleId = +match.params.id;
+    let isBookmarked;
+    if (bookmarkedArticles.length) {
+      isBookmarked = bookmarkedArticles.find(x => +x.id === currentViewedArticleId) ? true : null;
```

Comment:
What is x?

Reply:
@sulenchy feedback well received.
@sulenchy done.